packages/core/src/index.ts

```diff
@@ -43,6 +43,11 @@ interface PersistentStorage {
   set: (key: string, value: string) => void;
 }
 
+type flagCacheConfig = {
+  refetchFlags: boolean,
+  timeToLive: number
```

Comment:
maybe rename this thing into `timToLiveInSeconds` or `timeToLiveInHours` or whatever the unit is :)